### PR TITLE
Tweak my interests (and Mark Shannon's :-) in CODEOWNERS

### PR DESCRIPTION
**Pull request by gvanrossum**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -28,14 +28,17 @@ Objects/type*                 @markshannon
 Objects/codeobject.c          @markshannon
 Objects/frameobject.c         @markshannon
 Objects/call.c                @markshannon
-Python/ceval.c                @markshannon
+Python/ceval*.[ch]            @markshannon @gvanrossum
```
Comment (AlexWaygood) [inline .github/CODEOWNERS:31]: GitHub now says "This CODEOWNERS file contains errors" because of this line :(

Comment (gvanrossum) [reply .github/CODEOWNERS:31]: Whoops, sorry. Do you know what’s wrong? I am AFK.

Comment (AlexWaygood) [reply .github/CODEOWNERS:31]: I'll do a Google :-)

Comment (AlexWaygood) [reply .github/CODEOWNERS:31]: Looks like it's [this](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax):

> Warning: There are some syntax rules for gitignore files that *do not work* in CODEOWNERS files:
> 
> - Escaping a pattern starting with `#` using `\` so it is treated as a pattern and not a comment
> - Using `!` to negate a pattern
> - Using `[ ]` to define a character range

Comment (AlexWaygood) [reply .github/CODEOWNERS:31]: I'll make a PR to fix it

Comment (AlexWaygood) [reply .github/CODEOWNERS:31]: I filed #112210

**Pull request merged**